minio/README.md

```diff
@@ -0,0 +1,120 @@
+# Minio
+
+A Minio storage driver using [minio/minio](https://github.com/minio/minio).
+
+### Table of Contents
+- [Signatures](#signatures)
+- [Installation](#installation)
+- [Examples](#examples)
+- [Config](#config)
+- [Default Config](#default-config)
+
+### Signatures
+```go
+func New(config ...Config) Storage
+func (s *Storage) Get(key string) ([]byte, error)
+func (s *Storage) Set(key string, val []byte, exp time.Duration) error
+func (s *Storage) Delete(key string) error
+func (s *Storage) Reset() error
+func (s *Storage) Close() error
+func (s *Storage) CheckBucket() error
```

Comment:
These Bucket functions go against the `storage.go` interface

Reply:
Storage interface meets and adds additional features. There is an example in other storage enhancements.